AcadeMeData/migrations/0002_User_test_data.py

```diff
@@ -0,0 +1,25 @@
+from django.db import migrations, transaction
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('AcadeMeData', '0001_initial'),

```

Comment:
Please, it should be like this
    dependencies = [
        ('auth', '__latest__'),
        ('AcadeMeData', '0001_initial'),
    ]

Reply:
returns errors - please continue chat with me in slack

Reply:
@DanWaldman i think you should do some imports 
Please take a look here 
https://docs.djangoproject.com/en/3.2/topics/migrations/

Reply:
I think you missed the following imports
from django.db import migrations, transaction
from django.contrib.auth.models import User